File: CppSandBoxFalseSharing/ShareData.cpp


/** include
*/
#include "ShareData.h"
#if(1)


/** Simple
*/
namespace Simple
{
	/** LOOP_MAX
	*/
	#if(defined(_DEBUG))
	const int ShareData::LOOP_MAX = 100000;
	#else
	const int ShareData::LOOP_MAX = 300000;
	#endif
}

#endif

File: UnitySandBoxFalseSharing/Assets/ReadFromOtherThread/ShareData.cs


/** ReadFromOtherThread
*/
namespace ReadFromOtherThread
{
	/** ShareData
	*/
	public class ShareData
	{
		/** LOOP_MAX
		*/
		#if(UNITY_EDITOR)
		public const int LOOP_MAX = 20000;
		#else
		public const int LOOP_MAX = 80000;
		#endif

		/** value
		*/
		public System.UInt64 value;
	}
}


